### PR TITLE
Automatically build PHAR on release

### PR DESCRIPTION
**Pull request by dantleech**

### CODE REVIEW

File: .github/workflows/release-phar.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        run: test -e my127ws.phar && exit 0 || exit 10
+
+      - name: "Upload PHAR to Release"
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: my127ws.phar
+          asset_name: my127ws.phar
```
Comment (dantleech) [inline .github/workflows/release-phar.yml]: note it will attach as `my127ws.phar` vs. `ws` as with previous releases. Not sure if we care.

Comment (kierenevans) [reply .github/workflows/release-phar.yml]: Can we name it `ws` or do we want to handle that in the README instructions?

File: .github/workflows/release-phar.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+            fetch-depth: 0
+
+      - name: Set PHP 7.3
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.3'
```
Comment (andytson-inviqa) [inline .github/workflows/release-phar.yml:21]: I guess still up for discussion on whether to upgrade minimum version, but can change this along with whatever changes we do for the min version bump in another PR

File: .github/workflows/release-phar.yml (hunk 1 above)
Comment (dantleech) [reply .github/workflows/release-phar.yml]: i updated to rename it to `ws`

File: .github/workflows/release-phar.yml (hunk 2 above)
Comment (dantleech) [reply .github/workflows/release-phar.yml:21]: yeah

**Review by kierenevans — APPROVED** [review]

**Pull request merged**